ext/pycall/pycall.c

```diff
@@ -1270,6 +1280,7 @@ pycall_libpython_helpers_m_sequence_contains(VALUE mod, VALUE pyptr, VALUE key)
   if (res == -1) {
     pycall_pyerror_fetch_and_raise("PySequence_Contains");
   }
+  pycall_Py_DecRef(pyobj_key);
```

Comment:
Could you please move this line just after calling `PySequence_Contains`?

Reply:
OK